rollup-interface/src/state_machine/mocks.rs

```diff
@@ -156,6 +188,42 @@ impl Display for MockAddress {
 
 impl AddressTrait for MockAddress {}
 
+/// A trivial test validity condition structure that only contains a boolean
+#[derive(
+    Debug, BorshDeserialize, BorshSerialize, Serialize, Deserialize, PartialEq, Clone, Copy, Default,
+)]
+pub struct TestValidityCond {
+    /// The associated validity condition field. If it is true, the validity condition is verified
+    pub cond: bool,
```

Comment:
As struct already has `Condition` in its name I would suggest to reflect actual purpose of boolean attribute in its name:
```suggestion
    pub is_valid: bool,
```